examples/amp-ad/interstitial-creative.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html><html lang="en">
```

Comment:
Maybe this can be in an ads directory so that its clear its not a working amp example?